### DISCUSSION TITLE
How do I run scalene on large projects?

### DISCUSSION THREAD

**Discussion by ethindp** [plasma-umass/scalene#775] [Q&A]
I have a project created with briefcase. But when I try to use scalene, it can't load one of my relative imports (it thinks I'm trying to import `scalene.my_module` even though the import is `from .my_module import ...`). Am I doing something wrong?

**Comment by emeryberger** [comment]
Are you trying to programmatically invoke Scalene? The best way to use it is directly as a replacement for `python3`. I had not previously heard of briefcase and it may be doing something that conflicts with how Scalene operates.
